pylint/testutils/output_line.py

```diff
@@ -21,7 +21,7 @@ def __eq__(self, other):
             return self[:-1] == other[:-1]
         return NotImplemented  # pragma: no cover
 
-    def __hash__(self):
+    def __hash__(self):  # pylint:disable=invalid-hash-returned
         return None
```

Comment:
```suggestion
```
It should be fine to just remove it. From the python documentation:
```
A class that overrides __eq__() and does not define __hash__()
will have its __hash__() implicitly set to None.
```
https://docs.python.org/3/reference/datamodel.html#object.__hash__

Reply:
Yeah, I fixed that in #4077, can you do `git cherry-pick 293981fb130393dd310a75b601570907a4542387` ? We merged on master when the test were not passing and introduced additional errors.